resources/views/todos/index.blade.php

```diff
@@ -74,7 +81,22 @@
 
                                 <input type="submit" class='btn btn-danger' value='Delete'>
                             </form>
+                        <td>
+                            <div class="[ form-group ]">
+                                    <input type="checkbox" name="fancy-checkbox-success[]" id="fancy-checkbox-success" autocomplete="off" />
```

Comment:
why is this an array? Why did you call it fancy-checkbox can it not simply be `completed`. It think calling it completed makes more sense. You can still use class name for css